packages/main/src/StepInput.hbs

```diff
@@ -44,6 +44,11 @@
 			@focusout="{{_onInputFocusOut}}"
 			@focusin="{{_onInputFocusIn}}"
 	>
+
+		{{#if valueStateMessage.length}}
+			<slot name="valueStateMessage" slot="valueStateMessage" style="width: 100%"></slot>

```

Comment:
`style="width: 100%"` Is this have any effect on the slot element

Reply:
err, no... just tried something... will remove it now!